doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,101 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
+
+{{< admonition type="Disambiguation" >}}
+
+This page covers the usage of `ALTER CLUSTER` to change the cluster's
```

Comment:
With the new structure, we could actually fold in ALTER CLUSTER <cluster_name> RENAME TO in here.